src/Synology.Api.Client/Apis/DownloadStation/Info/Models/DownloadStationServerConfig.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System.Text.Json.Serialization;
+
+namespace Synology.Api.Client.Apis.DownloadStation.Info.Models
+{
+    public class DownloadStationServerConfig
+    {
+        public DownloadStationServerConfig(
```

Comment:
What's the expected behaviour when you set only one config and all the other values are null? Are the ones with null values ignored by the DSM?